### PR TITLE
Map observability-operator e2e test image for OCP z-stream test

### PR DESCRIPTION
**Pull request by lihongyan1**
used in PR
https://github.com/openshift/release/pull/42761

### CODE REVIEW

File: core-services/image-mirroring/observability-operator/OWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+approvers:
+  - lihongyan1
+options: {}
+reviewers:
+  - lihongyan1
```
Comment (bbguimaraes) [inline core-services/image-mirroring/observability-operator/OWNERS:5]: Please include more than a single owner.

**Comment by lihongyan1** [comment]
@bbguimaraes @smg247 Please help review

**Comment by lihongyan1** [comment]
/ pj-rehearse

**Comment by bbguimaraes** [comment]
> / pj-rehearse

@lihongyan1: =)

/pj-rehearse

**Pull request merged**